instagram/src/login.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
```

Comment:
Capitalize the name of this file for consistency, and it'll fix your import in authentication.js (capitalize that one too!)